views/includes/page-templates/trust-details.html

```diff
@@ -137,15 +137,15 @@ <h1 class="govuk-heading-m govuk-!-margin-0">Date the trust was created:</h1>
           {
             value: '1',
             text: 'Yes',
-            checked: formData.hasAllInfo == '1',
+            checked: formData.hasAllInfo == '0',
```

Comment:
This approach means that the radio buttons are populated the wrong way around. When you select "yes' and press save, then press back, "no' is populated.
When you select 'No' and press save, then press back, "yes' is populated.

Reply:
yes, in the bug, the boolean stored in Mongo is unable_to_obtain_all_trust_info == true

Reply:
but if you select yes on the page, go forwards, then back, the page says No because the 'checked:' value is opposite to 'value:'.
Similarly , when i selected No, my answer changed to Yes when i went back to the screen.

Reply:
I am unsure how else to resolve the bug as the declaration that is made during registration journey of unable_to_obtain_all_trust_info retrived during an update journey with current logic behaves opposite hence that solution.

Reply:
You'll need to:
1. Replicate the issue.
2. Ensure you test any changes locally before requesting a review.

Reply:
I'm still getting the problem of the radio button values flipping between leaving the page and going back. It's to do with the checked value being opposite the the radio button value.